### ISSUE TITLE
Websockets instructions do not work

### ISSUE THREAD

**Issue by Doesntmeananything** [luminus-framework/luminus#244]
Hi!

I'm new to clojure web development and I have followed all the instructions from the guide trying to create a websocket client/server. However, doing this in an existing project results in compilation errors. I have also checked the examples linked (both immutant and http-kit versions), and I can't `lein run` them either. Am I doing something terribly wrong, or is the code and examples indeed no longer work?

Thanks.

**Comment by yogthos** [comment]
The dependencies in the examples are somewhat out of date and won't work with JDK 9+. I'll take a look at updating them.

**Comment by yogthos** [comment]
Out of curiosity, what problems did you run into when following the instructions, that might be a separate issue.

**Comment by yogthos** [comment]
I've updated the [HTTP Kit example](https://github.com/luminus-framework/examples/tree/master/multi-client-ws-http-kit), let me know if it works for you.

**Comment by Doesntmeananything** [comment]
> Out of curiosity, what problems did you run into when following the instructions, that might be a separate issue.

Thanks a lot for your help!

When following the code examples, I got cryptic arity errors, and after some searching they seemed to point at an issue in transit-cljs: https://github.com/cognitect/transit-cljs/issues/49

I was really not sure how to proceed from there, so I decided to completely reinstall my JDK and... things worked. 🤦‍♂️

Whatever it was, it originated from my borked java installation. Your HTTP-kit example now compiles and works just fine. 

Thanks again for your help. It's a bit overwhelming to be trying to figure stuff out on my own, so any input is always extremely valuable.
